docs/usage.md

```diff
@@ -0,0 +1,86 @@
+# Usage
+
+Use the import tool from the command line, like so:
+```
+java -jar <path-to-jar> <mode> <list of mcf/tmcf/csv files>
+```
+
+Hint: it can be useful to create an alias for the jar file, such as:
+```
+alias dc-import='java -jar <path-to-jar>'
+```
+
+This is the form that will be used in the rest of the documentation.
+
+Hint: to access a concise explanation of usage modes and flags, run
+`dc-import --help`
+
+## Usage Modes
+
+The import tool can either lint StatVars and other schema files, or
+it can generate instance MCFs from a TMCF and CSV.
```

Comment:
This makes it like lint can only be used with SVs and schema files... how about:

In `lint` mode, the import tool validates the artifacts produced for addition to Data Commons.  These artifacts include [instance MCF files](link to mcf doc) and pairs of [TMCF](link to tmcf doc) and corresponding CSV files.

In `genmcf` mode, the import tool produces instance MCF files from a pair of TMCF file, and its associated CSV files. This mode subsumes `lint` command in the validations it performs.

All flags below apply to both the modes.